e2e/fiat_tf_test.go

```diff
@@ -1846,3 +1848,275 @@ func TestFiatTFBankSend(t *testing.T) {
+
+	gaiaWalletBal, err = gaia.GetBalance(ctx, gaiaWallet.FormattedAddress(), dstIbcDenom)
+	require.NoError(t, err)
+	require.Equal(t, transfer.Amount, gaiaWalletBal)
+}
+
+func TestFiatTFIBCIn(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+	t.Parallel()
+
+	ctx := context.Background()
+
+	nw, gaia, _, r, ibcPathName, _, eRep, _, _ := e2e.NobleSpinUpIBC(t, ctx, true)
+	noble := nw.Chain
+	val := noble.Validators[0]
+
+	w := interchaintest.GetAndFundTestUsers(t, ctx, "default", math.NewInt(1_000_000), noble, gaia)
+	nobleWallet := w[0] // 1_000_000ustake
+	gaiaWallet := w[1]  // 1_000_000uatom
+
+	mintAmount := int64(100)
+	_, err := val.ExecTx(ctx, nw.FiatTfRoles.Minter.KeyName(), "fiat-tokenfactory", "mint", nobleWallet.FormattedAddress(), fmt.Sprintf("%duusdc", mintAmount))
+	require.NoError(t, err, "error minting")
+
+	// noble -> gaia channel info
+	nobleToGaiaChannelInfo, err := r.GetChannels(ctx, eRep, noble.Config().ChainID)
+	require.NoError(t, err)
+	nobleToGaiaChannelID := nobleToGaiaChannelInfo[0].ChannelID
+	// gaia -> noble channel info
+	gaiaToNobleChannelInfo, err := r.GetChannels(ctx, eRep, gaia.Config().ChainID)
+	require.NoError(t, err)
+	gaiaToNobleChannelID := gaiaToNobleChannelInfo[0].ChannelID
+
+	amountToSend := math.NewInt(mintAmount)
+	transfer := ibc.WalletAmount{
+		Address: gaiaWallet.FormattedAddress(),
+		Denom:   e2e.DenomMetadataUsdc.Base,
+		Amount:  amountToSend,
+	}
+
+	// ibc transfer noble -> gaia
+	_, err = noble.SendIBCTransfer(ctx, nobleToGaiaChannelID, nobleWallet.KeyName(), transfer, ibc.TransferOptions{})
+	require.NoError(t, err)
+
+	// relay MsgRecvPacket & MsgAcknowledgement
+	require.NoError(t, r.Flush(ctx, eRep, ibcPathName, nobleToGaiaChannelID))
+
+	// uusdc IBC denom on gaia
+	srcDenomTrace := transfertypes.ParseDenomTrace(transfertypes.GetPrefixedDenom("transfer", gaiaToNobleChannelID, e2e.DenomMetadataUsdc.Base))
+	dstIbcDenom := srcDenomTrace.IBCDenom()
+
+	gaiaWalletBal, err := gaia.GetBalance(ctx, gaiaWallet.FormattedAddress(), dstIbcDenom)
+	require.NoError(t, err)
+	require.Equal(t, transfer.Amount, gaiaWalletBal)
+
+	// ACTION: IBC send in a TF token while TF is paused
+	// EXPECTED: Request fails;
+
+	e2e.PauseFiatTF(t, ctx, val, nw.FiatTfRoles.Pauser)
+
+	amountToSend = math.OneInt()
+	transfer = ibc.WalletAmount{
+		Address: nobleWallet.FormattedAddress(),
+		Denom:   dstIbcDenom,
+		Amount:  amountToSend,
+	}
+
+	height, err := noble.Height(ctx)
+	require.NoError(t, err)
+
+	tx, err := gaia.SendIBCTransfer(ctx, gaiaToNobleChannelID, gaiaWallet.KeyName(), transfer, ibc.TransferOptions{})
+	require.NoError(t, err, "error broadcasting IBC send")
+
+	require.NoError(t, r.Flush(ctx, eRep, ibcPathName, gaiaToNobleChannelID))
+
+	heightAfterFlush, err := noble.Height(ctx)
+	require.NoError(t, err)
+
+	ack, err := testutil.PollForAck(ctx, gaia, height, heightAfterFlush+5, tx.Packet)
+	require.NoError(t, err, "error polling for ack")
```

Comment:
@boojamya We are running into the encoding issue we were experiencing with conformance tests 🤔

Maybe we can simply wait 5 blocks after flush, and that will be sufficient?